### DISCUSSION TITLE
DynamicPageTitle & NavActions

### DISCUSSION THREAD

**Discussion by texttechne** [UI5/webcomponents-react#5085] [General]
Hi, 

the docs for the DynamicPageTitle state:
> The navigationActions position depends on the control size. If the control size is 1280px or bigger, they are rendered right next to the actions. Otherwise, they are rendered in the top-right area, above the actions.

In my scenario I have a flexible column layout, so I only get to >1280px with really big screens.
And I only need the close button in there. But having the close button in a separate row looks really unfortunate.

![image](https://github.com/SAP/ui5-webcomponents-react/assets/1963160/e33c221d-89a3-40dc-9a2a-1d09dd7db09a)

So my question: Is there a way to get the navActions float right to the title?
Since the HTML structure is changed...

**Comment by Lukas742** [comment]
Hi @texttechne 

changing this behavior and therefore forcing the nav actions to always be displayed in the same line is not supported. But since you want to display the "close" button within the Actions section, couldn't you just pass it to `actions`?

**Reply by texttechne** [reply]
works like a charm: Cheers!
